Sources/Data Model/Audience/Audience.swift

```diff
@@ -64,6 +64,21 @@ struct Audience: Codable, Equatable {
     }
     
     func evaluate(project: ProjectProtocol?, attributes: OptimizelyAttributes?) throws -> Bool {
-        return try conditions.evaluate(project: project, attributes: attributes)
+        let logger = OPTLoggerFactory.getLogger()
```

Comment:
A separate logger instance in each Audience must be a waste.
Let's create a single instance in Project and pass it as a parameter here:

func evaluate(project: ProjectProtocol?, attributes: OptimizelyAttributes?, logger: OPTLogger? = nil)

Reply:
Instantiating "OPTLogger" in "Project" will make "Project" less clean, but we can use the same logger for other logging later.